soda_machine.rb

```diff
@@ -7,12 +7,36 @@ def initialize(args = {})
   end
 
   def current_inventory_count
+    @sodas.size
   end
 
   def find_soda(soda_brand)
```

Comment:
would be better to use `each` here instead of looping using an index like this.  some students come from a Java background or some other programming language and kindof hold on to looping with an index.  it works but it's not idiomatic Ruby so you'll get dinged on it in any coding interview.  try looking at more open source Ruby code like what's in your local gem source files to see what professionally written Ruby looks like